src/mono/mono/mini/driver.c

```diff
@@ -1605,7 +1610,11 @@ mini_usage (void)
 		"\n"
 		"Development:\n"
 		"    --aot[=<options>]      Compiles the assembly to native code\n"
+#ifndef ENABLE_NETCORE
+		"    --debug=ignore    Disable debugging support, use --help-debug for details\n"
+#else
 		"    --debug[=<options>]    Enable debugging support, use --help-debug for details\n"
+#endif
```

Comment:
Note that the old --debug had all kinds of options to enable which have a perf cost like --debug=casts.

Reply:
It's still there - I messed up the printout.

Reply:
We should add the new option here too: https://github.com/dotnet/runtime/blob/a7f8cdc0de945561ae3fcaa72440c0ec56d69060/src/mono/mono/mini/driver.c#L1666-L1668